packer/templates/docker.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
Instead of copying this file, symlink it from common-packer. That way it will stay in sync with common-packer changes.

Reply:
Indeed! Done!